src/api/getSwapQuote.ts

```diff
@@ -37,8 +37,14 @@ export async function getSwapQuote(
     };
   }
   if (params.maxSlippage) {
+    let slippagePercentage = params.maxSlippage;
+    // Adjust slippage for V1 API (aggregator)
+    // V1 expects slippage in tenths of a percent (e.g., 30 = 3%)
+    if (params.useAggregator) {
+      slippagePercentage = (Number(params.maxSlippage) * 10).toString();
+    }
     apiParams = {
-      slippagePercentage: params.maxSlippage,
+      slippagePercentage: slippagePercentage,
       ...apiParams,
     };
```

Comment:
if we need this logic in a 3rd place, we should extract to a utility

Reply:
agreed